### PR TITLE
Fix for MATLAB root path displayed in Jenkins build logs.

### PR DESCRIPTION
**Pull request by abhargav-mw**
This fixes the unmatched slashes in the MATLAB root path

### CODE REVIEW

**Comment by mw-hrastega** [comment]
Hi, Anmol. The changes look good to me.

File: src/main/java/com/mathworks/ci/Utilities.java [hunk 1]
```diff
@@ -45,8 +45,9 @@ public static void addMatlabToEnvPathFrmAxis(Computer cmp, TaskListener listener
         }
 
         String matlabExecutablePath = getNodeSpecificHome(name,
-               cmp.getNode(), listener, env) + "/bin";
+               cmp.getNode(), listener, env) + ((Boolean.TRUE.equals(cmp.isUnix()))?"/bin" : "\\bin");
```
Comment (nbhoski) [inline src/main/java/com/mathworks/ci/Utilities.java:48]: Why do we need Boolean.TRUE.equal ? does't just cmp.isUnix() ? a : b works

**Review by nbhoski — APPROVED** [review]
LGTM , with some minor comment. re run azure tc once i push my changes in.

File: src/main/java/com/mathworks/ci/Utilities.java (hunk 1 above)
Comment (abhargav-mw) [reply src/main/java/com/mathworks/ci/Utilities.java:48]: isUnix() has @ CheckForNull annotation and may return null pointer exception, This is a fail-safe

**Review by acampbel — APPROVED** [review]

**Pull request merged**